docs/cli/init.md

```diff
@@ -7,7 +7,13 @@ sidebar_position: 3
 
 Initiate a repository to start using packit. By default this command adds
 `.packit.yaml` config file to the git repository in the current working
-directory.  
+directory.
+
+If a `.pre-commit-config.yaml` file is found at the root of the git repository, `init` will add a pre-commit hook, which automatically runs `packit config validate` to check the validity of `.packit.yaml` upon pre-commit. Before adding this hook, `init` also checks for the presence of a `.git` repo. This behavior can be disabled using the `--without-precommit` flag.
+
+In case `.pre-commit-config.yaml` is not found at the rooot of the repository, this behavior is skipped and can be forced using the `--force-precommit` flag. Please note that the
```

Comment:
```suggestion
In case `.pre-commit-config.yaml` is not found at the root of the repository, this behavior is skipped and can be forced using the `--force-precommit` flag. Please note that the
```